sfs/array.py

```diff
@@ -15,7 +15,7 @@
 
 
 class ArrayData(namedtuple('ArrayData', 'x n a')):
-    """Named tuple returned by array functions.
+    """Get named tuple returned by array functions.
```

Comment:
This is not a function, therefore the docstring was more appropriate before.

Reply:
fixed